src/VIPServer.java

```diff
@@ -107,24 +107,6 @@ public void main(String[] args) throws HostFailureException, HostNotFoundExcepti
 					// SHOULD BE REPLACED BY
 					// job.carryOn();
 				} else {
-					if (!timer) {
```

Comment:
not so long term ... before the end of the PhD would be great ;) 
I'm ok to skip it for now, but it bothers me to have a half-baked support of computing time in the code.